weblate/addons/models.py

```diff
@@ -249,8 +260,16 @@ def handle_addon_event(
                         addon.name,
                     )
                     addon.disable()
-            else:
-                scope.log_debug("completed %s add-on: %s", event.label, addon.name)
+            finally:
+                if event.label not in exclude_from_logging:
+                    AddonActivityLog.objects.create(
+                        addon=addon,
+                        component=component,
+                        event=event,
+                        details={"result": log_result, "error": error_occurred},
+                    )
+                else:
+                    scope.log_debug("completed %s add-on: %s", event.label, addon.name)
```

Comment:
The system logging should stay:

```suggestion
                scope.log_debug("completed %s add-on: %s", event.label, addon.name)
```